tests/Microsoft.Identity.Test.Unit.net45/CoreTests/WsTrustTests/WsTrustEndpointTests.cs

```diff
@@ -164,6 +165,39 @@ public void TestWsTrustEndpointUsernamePasswordWsTrust13()
             Assert.AreEqual(expectedMessage, requestMessage);
         }
 
+        [TestMethod]
+        [Description("Assert client info is included in userrealm discovery header")]
+        public void UserrealmDiscoveryHeadersTest()
+        {
+            string endpoint = "https://login.microsoftonline.com/common/userrealm/";
```

Comment:
Consider using the //arrange //act //assert template for making long tests, such as this one, more readable